src/elexmodel/models/BaseElectionModel.py

```diff
@@ -234,7 +226,9 @@ def get_unit_prediction_interval_bounds(self, reporting_units, nonreporting_unit
         conformalization_data["lower_bounds"] = conformalization_lower_bounds
 
         # apply lower/upper models to nonreporting data
-        nonreporting_units_features = nonreporting_units[self.features].values
+        # now the features of teh nonreporting_units will be the same as the train_data features
+        # they might differ slightly from the features used when fitting the median prediction
+        nonreporting_units_features = self.featurizer.featurize_heldout_data(nonreporting_units)
```

Comment:
Just want to clarify: this step now means the nonreporting units (which previously had their fixed effects matched to the reporting units) is now having them re-matched to the training/comform fixed effects. So it is still the case that for a non-reporting unit, the fixed effects used to get its point prediction can be different from those used to get its 5th/95th percentile points, correct? I dont think thats necessarily bad, just want to confirm this is what you intended.

Reply:
Yes, but this was previously already the case. We never made sure that the train data had the same columns as the reporting data, so it was always possible that the model was slightly different.

Reply:
cool